docs/status/build

```diff
@@ -1,8 +1,24 @@
 #!/usr/bin/env python
 
+"""Build the status page.
+
+To test manually, you'll need to set up a "personal access token" with
+read permissions. On CI we use `GITHUB_TOKEN`.
+
+Usage:
+  $ export GITHUB_TOKEN=<token>
```

Comment:
consider adding `$ cd docs/status`?

Reply:
Is intended to be run from this directory.